### ISSUE TITLE
Webpack error

### ISSUE THREAD

**Issue by brandondurham** [smartystreets/smartystreets-javascript-sdk-utils#12]
I’m working in a component library that uses [Storybook](https://storybook.js.org/). When I try to spin up Storybook locally (`yarn storybook`), it errors out with this message:

```bash
ERROR in ./node_modules/smartystreets-javascript-sdk-utils/src/isInvalid.js 21:25
Module parse failed: Unexpected token (21:25)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| 	if (exactlyOneResult) {
> 		if (result[0].analysis?.enhanced_match) {
| 			const enhancedMatch = result[0].analysis.enhanced_match.split(",");
| 			return enhancedMatch.includes("none");
 @ ./node_modules/smartystreets-javascript-sdk-utils/index.js 3:12-38
 @ ./src/components/FormAddress/index.js
 @ ./src/components/AccountSettings/index.js
 @ ./src/components/AccountSettings/README.stories.mdx
 @ ./src sync ^\.(?:(?:^|\/|(?:(?:(?!(?:^|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.mdx)$
 @ ./.storybook/generated-stories-entry.js
 @ multi ./node_modules/@storybook/core-client/dist/esm/globals/polyfills.js ./node_modules/@storybook/core-client/dist/esm/globals/globals.js ./.storybook/storybook-init-framework-entry.js ./node_modules/@storybook/addon-docs/dist/esm/frameworks/common/config.js-generated-other-entry.js ./node_modules/@storybook/addon-docs/dist/esm/frameworks/react/config.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/esm/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-actions/dist/esm/preset/addArgs.js-generated-other-entry.js ./node_modules/@storybook/addon-backgrounds/dist/esm/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-backgrounds/dist/esm/preset/addParameter.js-generated-other-entry.js ./node_modules/@storybook/addon-measure/dist/esm/preset/preview.js-generated-other-entry.js ./node_modules/storybook-addon-outline/dist/esm/preset/addDecorator.js-generated-other-entry.js ./node_modules/@storybook/addon-a11y/dist/esm/a11yRunner.js-generated-other-entry.js ./node_modules/@storybook/addon-a11y/dist/esm/a11yHighlight.js-generated-other-entry.js ./node_modules/@storybook/addon-links/dist/esm/preset/addDecorator.js-generated-other-entry.js ./.storybook/preview.js-generated-config-entry.js ./.storybook/generated-stories-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=false&noInfo=undefined
```

Any idea what might be going on?

**Comment by brandondurham** [comment]
It’s because the optional chaining isn’t being transformed. If I change this:
```js
if (result[0].analysis?.enhanced_match) {
```
… to this:
```js
if (result[0].analysis && result[0].analysis.enhanced_match) {
```
… it works.

**Comment by camiblanch** [comment]
I think in this case you will need to update storybook and/or update babel and potentially your babel plugins. Optional chaining is well supported to my knowledge

**Comment by camiblanch** [comment]
After further investigation and trial, having optional chaining breaks things for our use case as well, so I will be updating things to no longer use optional chaining

**Comment by camiblanch** [comment]
Solved by 72209cf60a1bfa6c87b5a3b5ef801f5e40203941

Publishing patch now...

**Comment by camiblanch** [comment]
Published to version 1.2.1

**Comment by brandondurham** [comment]
NICE!!!! Yeah, I didn’t want to have to create a custom babel/webpack config for this one line, so I forked it and changed it, but you know what? This is better! Thank you.
